src/_igraph/dfsiter.c

```diff
@@ -0,0 +1,295 @@
+/* -*- mode: C -*-  */
+/* 
+   IGraph library.
+   Copyright (C) 2006-2012  Tamas Nepusz <ntamas@gmail.com>
```

Comment:
Please update the copyright note to year 2020, and "The igraph development team". I think that for each source file it is enough to start the copyright from the year that file was added.

Reply:
fixed